api/login.go

```diff
@@ -25,22 +28,29 @@ func (cli *Client) LoginWithGithub(endpoint string, githubToken string) (*User,
 	return userResp.User, nil
 }
 
-func (cli *Client) LoginWithVault(endpoint string, vaultToken string) (*User, error) {
-	url := endpoint + pathPrefix + "/login"
-	req, err := http.NewRequest("POST", url, nil)
+func (cli *Client) LoginWithVault(endpoint string, vault_url string, githubToken string) (*User, error) {
+
+	req, err := http.NewRequest("POST",
+		vault_url+"/v1/auth/github/login",
+		strings.NewReader("{\"token\":\""+githubToken+"\"}"))
 	if err != nil {
 		return nil, err
 	}
-	req.Header.Add("X-Vault-Token", vaultToken)
 	b, err := cli.rawRequest(req)
```

Comment:
variable called `b` doesn't make sense to me

Reply:
Made it make more sense